zhaquirks/xiaomi/custom/z03mmc.py

```diff
@@ -0,0 +1,156 @@
+"""Xiaomi LYWSD03MMC Bluetooth temperature and humidity sensor."""
+
+from zigpy.profiles import zha
+from zigpy.types import Bool, int16s, uint16_t
+from zigpy.zcl.clusters.general import Basic, Identify, Ota, PowerConfiguration
+from zigpy.zcl.clusters.hvac import UserInterface as UserInterfaceBase
+from zigpy.zcl.clusters.measurement import (
+    RelativeHumidity as RelativeHumidityBase,
+    TemperatureMeasurement as TemperatureMeasurementBase,
+)
```

Comment:
Can we keep the original name for the ZCL imports and just name the custom clusters something like `RelativeHumidityCustom`?

Reply:
Sure, that works. I named the derived class so that it matches the base class name and is transparent in the homeassistant UI, but if this naming scheme is preferred then that works for me.